.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+#### Includes new or updated dependencies?
```

Comment:
Are we moving away from including things like "What does this PR do", "Why are these changes being introduced", "Relevant tickets" in the PR template? I particularly find the "How can a reviewer manually see the effects of these changes?" section to be super useful.

Reply:
Great question.
 "What does this PR do", "Why are these changes being introduced", "Relevant tickets" should all be part of the commit message and thus shouldn't need to be called out in the PR template.

"How can a reviewer manually see the effects of these changes?" should probably return to the templates.